indigo_api/models/documents.py

```diff
@@ -183,21 +183,38 @@ def add_ids(toc):
     def commenceable_provisions(self):
```

Comment:
I think an outcome of our discussion is that all these methods are removed in favour of being run on the work object, since logically they belong there and not here.